### PR TITLE
Upgrade vtk and vcs-js.

### PR DESCRIPTION
**Pull request by danlipsa**
Also fix VTK build on Ubuntu 16.04

### CODE REVIEW

**Comment by danlipsa** [comment]
@doutriaux1 Would you please build VTK and vcs-js. Also test vcs with the new VTK. Thanks!

**Comment by doutriaux1** [comment]
will do. Thank YOU for doing this!

**Comment by doutriaux1** [comment]
@danlipsa I guess we should also bring the VTK SHA1 into or UV-CDAT VTK clone, no? So that we can tag it 2.12

**Comment by danlipsa** [comment]
@doutriaux1 Isn't that the same with the VTK used by vcs?

**Comment by danlipsa** [comment]
Is there a different place besides conda-recipes where we specify the SHA?

**Comment by doutriaux1** [comment]
well... We have a clone of VTK on our UV-CDAT organization and I merge your sha1 into this so I can tag it with a name instead of just a SHA1.

**Comment by doutriaux1** [comment]
it doesn't affect conda, just for us to remember exactly with SHA1 matches which version.

File: vtk-cdat/build.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ else
         export LD_LIBRARY_PATH=${PREFIX}/lib:${LIBRARY_PATH}
         export CC="gcc -Wl,-rpath=${PREFIX}/lib"
         export CXX="g++ -Wl,-rpath=${PREFIX}/lib"
-        export LDFLAGS="-lm"
+        export LDFLAGS="-L/usr/lib/x86_64-linux-gnu -lm"
```
Comment (doutriaux1) [inline vtk-cdat/build.sh]: @danlipsa which library is in this directory that wasn't found otherwise? Should we try to bring it in via conda?

**Pull request merged**